============================= test session starts ==============================
platform linux -- Python 3.10.12, pytest-8.3.3, pluggy-1.5.0
rootdir: /home/james/projects/cs6223/our_proptest_data
plugins: hypothesis-6.112.2, timeout-2.3.1
collected 25 items

doc_only/proptest/decimal.Decimal.exp/pbt_1.py F.FFF                     [ 20%]
doc_only/proptest/decimal.Decimal.exp/pbt_2.py FFF.F                     [ 40%]
doc_only/proptest/decimal.Decimal.exp/pbt_3.py F.EFF                     [ 60%]
doc_only/proptest/decimal.Decimal.exp/pbt_4.py F.EFF                     [ 80%]
doc_only/proptest/decimal.Decimal.exp/pbt_5.py F.EFF                     [100%]

==================================== ERRORS ====================================
_____________ ERROR at setup of test_strictly_increasing_property ______________
file /home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_3.py, line 14
  @given(st.floats(allow_nan=False, allow_infinity=False))
  def test_strictly_increasing_property(x1, x2):
E       fixture 'x1' not found
>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_3.py:14
______________ ERROR at setup of test_output_increasing_property _______________
file /home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_4.py, line 14
  @given(st.floats(allow_nan=False, allow_infinity=False))
  def test_output_increasing_property(x1, x2):
E       fixture 'x1' not found
>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_4.py:14
____________ ERROR at setup of test_exponential_increasing_property ____________
file /home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_5.py, line 14
  @given(st.floats(allow_nan=False, allow_infinity=False))
  def test_exponential_increasing_property(x1, x2):
E       fixture 'x1' not found
>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_5.py:14
=================================== FAILURES ===================================
/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_1.py:8: exceptiongroup.ExceptionGroup: Hypothesis found 2 distinct failures. (2 sub-exceptions)
/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_1.py:22: decimal.Overflow: [<class 'decimal.Overflow'>]
/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_1.py:28: AssertionError: assert Decimal('1') < 1
/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_1.py:32: decimal.Overflow: [<class 'decimal.Overflow'>]
/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_2.py:8: exceptiongroup.ExceptionGroup: Hypothesis found 4 distinct failures. (4 sub-exceptions)
/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_2.py:14: hypothesis.errors.InvalidArgument: Too many positional arguments for test_output_equals_one_when_input_is_zero_property() were passed to @given - expected at most 0 arguments, but got 1 (decimals(),)
/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_2.py:18: exceptiongroup.ExceptionGroup: Hypothesis found 3 distinct failures. (3 sub-exceptions)
/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_2.py:33: AssertionError: assert Decimal('1') > 1e+50
/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_3.py:5: exceptiongroup.ExceptionGroup: Hypothesis found 2 distinct failures. (2 sub-exceptions)
/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_3.py:24: AssertionError: assert Decimal('1') < 1
/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_3.py:28: decimal.Overflow: [<class 'decimal.Overflow'>]
/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_4.py:5: exceptiongroup.ExceptionGroup: Hypothesis found 2 distinct failures. (2 sub-exceptions)
/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_4.py:24: AssertionError: assert Decimal('1') < 1
/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_4.py:28: decimal.Overflow: [<class 'decimal.Overflow'>]
/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_5.py:5: exceptiongroup.ExceptionGroup: Hypothesis found 2 distinct failures. (2 sub-exceptions)
/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_5.py:24: AssertionError: assert Decimal('1') < 1
/home/james/projects/cs6223/our_proptest_data/doc_only/proptest/decimal.Decimal.exp/pbt_5.py:28: decimal.Overflow: [<class 'decimal.Overflow'>]
=========================== short test summary info ============================
FAILED doc_only/proptest/decimal.Decimal.exp/pbt_1.py::test_exp_output_is_positive_property
FAILED doc_only/proptest/decimal.Decimal.exp/pbt_1.py::test_exp_is_strictly_increasing_property
FAILED doc_only/proptest/decimal.Decimal.exp/pbt_1.py::test_exp_approaches_zero_as_input_negative_infinity_property
FAILED doc_only/proptest/decimal.Decimal.exp/pbt_1.py::test_exp_grows_without_bound_as_input_positive_infinity_property
FAILED doc_only/proptest/decimal.Decimal.exp/pbt_2.py::test_output_is_positive_property
FAILED doc_only/proptest/decimal.Decimal.exp/pbt_2.py::test_output_equals_one_when_input_is_zero_property
FAILED doc_only/proptest/decimal.Decimal.exp/pbt_2.py::test_output_is_strictly_increasing_property
FAILED doc_only/proptest/decimal.Decimal.exp/pbt_2.py::test_output_tends_to_infinity_for_positive_infinity_property
FAILED doc_only/proptest/decimal.Decimal.exp/pbt_3.py::test_positive_output_property
FAILED doc_only/proptest/decimal.Decimal.exp/pbt_3.py::test_approaches_zero_negative_infinity_property
FAILED doc_only/proptest/decimal.Decimal.exp/pbt_3.py::test_approaches_infinity_positive_infinity_property
FAILED doc_only/proptest/decimal.Decimal.exp/pbt_4.py::test_output_positive_property
FAILED doc_only/proptest/decimal.Decimal.exp/pbt_4.py::test_output_approaching_zero_property
FAILED doc_only/proptest/decimal.Decimal.exp/pbt_4.py::test_output_tending_to_infinity_property
FAILED doc_only/proptest/decimal.Decimal.exp/pbt_5.py::test_exponential_output_positive_property
FAILED doc_only/proptest/decimal.Decimal.exp/pbt_5.py::test_exponential_negative_infinity_property
FAILED doc_only/proptest/decimal.Decimal.exp/pbt_5.py::test_exponential_positive_infinity_property
ERROR doc_only/proptest/decimal.Decimal.exp/pbt_3.py::test_strictly_increasing_property
ERROR doc_only/proptest/decimal.Decimal.exp/pbt_4.py::test_output_increasing_property
ERROR doc_only/proptest/decimal.Decimal.exp/pbt_5.py::test_exponential_increasing_property
==================== 17 failed, 5 passed, 3 errors in 4.17s ====================
